tests/Unit/Form/Type/SeoMetadataTypeTest.php

```diff
@@ -14,24 +14,42 @@
 use Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SeoMetadata;
 use Symfony\Cmf\Bundle\SeoBundle\Form\Type\SeoMetadataType;
 use Symfony\Cmf\Bundle\SeoBundle\Model\SeoMetadata as SeoMetadataModel;
+use Symfony\Component\Form\Extension\Validator\ValidatorExtension;
 use Symfony\Component\Form\PreloadedExtension;
-use Symfony\Component\Form\Tests\Extension\Validator\Type\TypeTestCase;
+use Symfony\Component\Form\Test\FormIntegrationTestCase;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 /**
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
  */
-class SeoMetadataTypeTest extends TypeTestCase
+class SeoMetadataTypeTest extends FormIntegrationTestCase
 {
+    private $validator;
+
+    public function setUp()
+    {
+        $this->validator = $this->createMock(ValidatorInterface::class);
```

Comment:
we have to set it up on our own, as the base class in gone (moved to a trait) in symfony 3.4